compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -21,6 +21,38 @@ static const char *kGpuModuleImportPath =
 namespace py = pybind11;
 using namespace mlir::python::adaptors;
 
+py::list ireeCodegenGetExecutableVariantOpsBinding(MlirModule module) {
+  int numOps = 0;
+  ireeCodegenGetExecutableVariantOps(module, &numOps, nullptr);
+
+  std::vector<MlirOperation> ops(numOps);
+
+  ireeCodegenGetExecutableVariantOps(module, &numOps, ops.data());
+
+  py::list opsList;
+  for (int i = 0; i < numOps; ++i) {
+    opsList.append(ops[i]);
+  }
+
+  return opsList;
+}
+
+py::list ireeCodegenQueryMMAIntrinsicsBinding(MlirOperation op) {
+  int numMMAs = 0;
+  ireeCodegenQueryMMAIntrinsics(op, &numMMAs, nullptr);
+
+  std::vector<uint32_t> mmaIntrinsics(numMMAs);
+
+  ireeCodegenQueryMMAIntrinsics(op, &numMMAs, mmaIntrinsics.data());
+
+  py::list opsList;
+  for (int i = 0; i < numMMAs; ++i) {
+    opsList.append(mmaIntrinsics[i]);
+  }
+
+  return opsList;
```

Comment:
Here, we should return a list of enums, not integers. You can see how to construct an enum in the code that handles enum attributes below.